src/features/student/locales/fr.json

```
@@ -364,8 +357,19 @@
             "success": "Votre trace a été ajoutée à votre bibliothèque."
           }
         },
-        "studentToolsTracesActionButtons": {
-          "addTrace": "AJOUTER UNE TRACE DANS MA BIBLIOTHÈQUE"
+        "studentToolsTracesViewContainer": {
+          "pagination": {
+            "bottom": {
+              "ariaLabel": "Pagination basse"
+            },
+            "top": {
+              "ariaLabel": "Pagination haute"
+            }
+          }
+        },
+        "studentToolsTracesViewTabs": {
+          "associated": "Mes traces associées ({count})",
+          "unassociated": "Mes traces non associées ({count})"
```

```suggestion
          "associated": "Aucune trace associée ({count})  | Ma trace associée ({count}) | Mes traces associées ({count})",
          "unassociated": "Aucune trace non associée ({count})  | Ma trace non associée ({count}) | Mes traces associées ({count})"
```
Je pensais à un truc comme ça, à voir si la parenthèse sert dans les deux premiers cas